modules/core/exported/client.go

```diff
@@ -56,8 +56,10 @@ type ClientState interface {
 	// Genesis function
 	ExportMetadata(sdk.KVStore) []GenesisMetadata
 
-	// Update and Misbehaviour functions
+	// VerifyClientMessage verifies a Header or a Misbehaviour
```

Comment:
```suggestion
	// VerifyClientMessage verifies a ClientMessage. A ClientMessage could be a Header, Misbehaviour, or batch update.
```
we should probably add more description here

Reply:
Yeah much better thanks